.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -226,8 +257,49 @@
         </ItemGroup>
     </Target>
 
+    <Target Name="ProcessMauiSplashScreens"
+        Condition="'@(MauiSplashScreen)' != ''"
```

Comment:
Should we look for > 1 and error, or at least warn if there's more than one item in the group?

Reply:
I made it a warning, because the app still worked fine.